reanalysis/wrapper.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+
+
+"""
+wrapper for reanalysis process
+
+currently still reliant on a dataproc cluster for the Hail-VEP runtime environment
+
+compound-het calculations moving to Hail, removing the call to Clinvar
+"""
+
+
+from typing import Any, Dict, Optional, Union
+
+import json
+import logging
+import os
+import sys
+
+import click
+from cloudpathlib import AnyPath
+import hailtop.batch as hb
+
+from cpg_utils.hail import init_batch, output_path
+
+from query_panelapp import main as panelapp_main
+from hail_filter_and_categorise import main as category_main
+
+
+DEFAULT_IMAGE = os.getenv('CPG_DRIVER_IMAGE')
+assert DEFAULT_IMAGE
+
+# static paths to write outputs
+PANELAPP_JSON_OUT = output_path('panelapp_137_data.json')
+HAIL_VCF_OUT = output_path('hail_categorised.vcf.bgz')
+COMP_HET_JSON = output_path('hail_comp_het.json')
+CONFIG_OUT = output_path('config_used.json')
+REHEADERED_OUT = output_path('hail_categories_reheadered.vcf.bgz')
+MT_TMP = output_path('tmp_hail_table.mt', category='tmp')
+
+# location of the Slivar Docker image
+AR_REPO = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
```

Comment:
We should probably fetch that from an environment variable, too. I can prepare that for the analysis-runner.